### PR TITLE
Watch feature and tests

### PR DESCRIPTION
**Pull request by cronan03**
Command is working currently

### CODE REVIEW

**Comment by fcollonval** [comment]
Hey @cronan03 could execute the linter on the PR: `python -m pre_commit run -a`

File: jupyter_builder/extension_commands/watch.py [hunk 1]
```diff
@@ -19,7 +20,8 @@ class WatchLabExtensionApp(BaseExtensionApp):
     source_map = Bool(False, config=True, help="Generate source maps")
 
     core_path = Unicode(
-        os.path.join(HERE, "staging"),
+        # os.path.join(HERE, "staging"),
```
Comment (fcollonval) [inline jupyter_builder/extension_commands/watch.py:23]: Let's drop unused code

```suggestion
```

File: jupyter_builder/federated_extensions.py [hunk 2]
```diff
@@ -227,7 +227,8 @@ def watch_labextension(
     path, labextensions_path, logger=None, development=False, source_map=False, core_path=None
 ):
     """Watch a labextension in a given path"""
-    core_path = osp.join(HERE, "staging") if core_path is None else str(Path(core_path).resolve())
+    # core_path = osp.join(HERE, "staging") if core_path is None else str(Path(core_path).resolve())
```
Comment (fcollonval) [inline jupyter_builder/federated_extensions.py:230]:
```suggestion
```

File: jupyter_builder/jlpm.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+"""A Jupyter-aware wrapper for the yarn package manager"""
```
Comment (fcollonval) [inline jupyter_builder/jlpm.py:1]: It is a good practice to not mix up changes in a PR to focus on one bug fix or feature.

But let's keep it this time

**Review by fcollonval — CHANGES_REQUESTED** [review]
Thanks @cronan03 

Minor suggestions but overall it looks good.

**Comment by fcollonval** [comment]
For the falling tests, could you execute the auto fix `python -m pre_commit run -a`

For the failing windows test, could you try increasing the sleep interval before checking the files changes. It may be all that is needed to fix the test.

File: jupyter_builder/jlpm.py (hunk 3 above)
Comment (cronan03) [reply jupyter_builder/jlpm.py:1]: Okay thanks @fcollonval , will keep that in mind

**Comment by fcollonval** [comment]
Hey @cronan03 could you please not add the yarn.js file in this PR and create a new one for it?

**Review by fcollonval — APPROVED** [review]
Yeah let's merge

**Pull request merged**